src/utils/navigation.ts

```diff
@@ -25,6 +25,11 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.UPDATE_LANDING_PAGE_URL,
     nextPage: [config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE]
   },
+  [config.UPDATE_CONFIRMATION_URL]: {
+    currentPage: config.UPDATE_CONFIRMATION_PAGE,
+    previousPage: () => config.OVERSEAS_ENTITY_QUERY_URL,
+    nextPage: [config.UPDATE_LANDING_PAGE_URL]
```

Comment:
previousPage and nextPage seem incorrect. I think nextPage can just be [] if there is nothing after your page. previousPage not sure what should be